src/components/formProducts/FormProducts.js

```diff
@@ -99,19 +97,40 @@ const FormProducts = ({ items }) => {
         </div>
         <div className="productos_view products-container">
           {itemsFiltered?.length > 0 ? (
-            itemsFiltered.map((item, index) => (
-              <ProductForList
-                key={`${index}-${item.name}`}
-                item={item}
-                token={token.current}
+            <>
+              <ProductsByEstimation
+                items={itemsFiltered}
+                start={0}
+                end={7}
```

Comment:
Valores de las propiedades `start` and `end` **no coincide**n con el AC definido en el [issue 12](https://github.com/the-collab-lab/tcl-39-smart-shopping-list/issues/12).

>  Necesita comprarlo pronto (menos de 7 días)

![image](https://user-images.githubusercontent.com/8689897/158035394-cc617093-88fd-49a6-a9b8-8d00761e21b6.png)

Por favor, te agradecería su cambio:

```suggestion
                start={0}
                end={6}
```